src/formElementPlugins/longText/src/ApplicationView.tsx

```diff
@@ -14,12 +14,15 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
   onSave,
   Markdown,
 }) => {
-  const { label, description, placeholder, lines, maxLength } = parameters
+  const { label, description, placeholder, lines, default: defaultValue, maxLength } = parameters
 
   const { isEditable } = element
 
   useEffect(() => {
-    onUpdate(value)
+    if (!value && defaultValue) {
+      onSave({ text: defaultValue })
+      setValue(defaultValue)
```

Comment:
I am getting confused now, how the ApplicationViewWrapper interacts with ApplciationView and useGetFullApplicationStructure, sound like we need another diagram : /

I am not sure what the difference here is between calling onUpdate and not calling it

Reply:
onUpdate just updates the local state "value", whereas onSave saves it to the database. However, onSave *includes* a call to onUpdate within it, so no need to do onUpdate if you're doing onSave :)